src/pages/index.tsx

```diff
@@ -61,6 +72,38 @@ const Home: NextPage = () => {
     return days < 30;
   };
 
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const query = event.target.value;
+    setIsSearching(query);
+
+    if (query.length >= 3) {
+      performSearch(query.toLowerCase());
+    } else {
+      setSearchResults([]);
+    }
+  
+    function performSearch(query: string) {
```

Comment:
the convention in this app is using arrow funcs - probably need an eslint rule to enforce as well